Code/JungleHelper.csproj

```diff
@@ -90,7 +77,7 @@
     <Reference Include="System.Data" />
     <Reference Include="System.Xml" />
     <Reference Include="YamlDotNet, Version=8.0.0.0, Culture=neutral, PublicKeyToken=ec19458f3c15af5e, processorArchitecture=MSIL">
-      <HintPath>..\packages\YamlDotNet.8.1.2\lib\net45\YamlDotNet.dll</HintPath>
+      <HintPath>..\..\..\YamlDotNet.dll</HintPath>

```

Comment:
Those changes might break the auto builds, I'll revert them once this is ready to be merged

Reply:
Understood, I didn't edit that on my end so I assume it's a weird preferences thing. (Gonna marked this as resolved since it's not something I can edit given my skills atm.)

Reply:
maybe you just messed with project references to fix them on your end 😅